### PR TITLE
RDoc-2403_ShardingUnsupported

### PR DESCRIPTION
**Pull request by reebhub**

### CODE REVIEW

File: Documentation/6.0/Raven.Documentation.Pages/sharding/unsupported.markdown [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+| ------------- | ------------- | 
+| [Filtered Replication](../studio/database/tasks/ongoing-tasks/hub-sink-replication/overview#filtered-replication) |  | 
+| [Hub/Sink Replication](../studio/database/tasks/ongoing-tasks/hub-sink-replication/overview) |  | 
+| **Legacy replication** | From RavenDB 3.x instances | 
+
+## Other Unsupported Features
+
+| Unsupported Feature | Comment | 
+| ------------- | ------------- | 
+| [Tombstones cleanup](../glossary/tombstone#tombstone-cleaning) | By POST, Studio |
```
Comment (karmeli87) [inline Documentation/6.0/Raven.Documentation.Pages/sharding/unsupported.markdown:123]: Let's remove this. This is supported per shard, just need to adjust the studio

File: Documentation/6.0/Raven.Documentation.Pages/sharding/unsupported.markdown [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+| [Migrate from RavenDB](../studio/database/tasks/import-data/import-from-ravendb) | By POST, Studio, smuggler | 
+| [Migrate from SQL DB](../studio/database/tasks/import-data/import-from-sql) |  | 
+
+## Unsupported Subscriptions Features
+
+| Unsupported Feature | Comment | 
+| ------------- | ------------- | 
+| [Concurrent Subscriptions](../client-api/data-subscriptions/concurrent-subscriptions) |  | 
+
+## Unsupported Debug Features
```
Comment (karmeli87) [inline Documentation/6.0/Raven.Documentation.Pages/sharding/unsupported.markdown:91]: I would prefer to remove this category.
Most are supported for the single shard.

File: Documentation/6.0/Raven.Documentation.Pages/sharding/unsupported.markdown [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+| [MoreLikeThis](../client-api/session/querying/how-to-use-morelikethis) |  | 
+| **Order by score** |  | 
+
+## Unsupported Document Extensions Features
+
+| Unsupported Feature | Comment | 
+| ------------- | ------------- | 
+| **Move Attachments** | E.g. `session.Advanced.Attachments.Move("users/1","foo","users/2","bar");` is not supported. | 
+| **Copy Attachments** | E.g. `session.Advanced.Attachments.Copy("users/1","foo","users/2","bar");` is not supported. | 
+| **Get multiple Attachments** | E.g. `session.Advanced.Attachments.Get(attachmentNames)` is not supported. |
```
Comment (karmeli87) [inline Documentation/6.0/Raven.Documentation.Pages/sharding/unsupported.markdown:60]: Hmm this surprising tbh, need to check if this indeed the case :)

File: Documentation/6.0/Raven.Documentation.Pages/sharding/unsupported.markdown (hunk 1 above)
Comment (reebhub) [reply Documentation/6.0/Raven.Documentation.Pages/sharding/unsupported.markdown:123]: done

File: Documentation/6.0/Raven.Documentation.Pages/sharding/unsupported.markdown (hunk 2 above)
Comment (reebhub) [reply Documentation/6.0/Raven.Documentation.Pages/sharding/unsupported.markdown:91]: removed

File: Documentation/6.0/Raven.Documentation.Pages/sharding/unsupported.markdown (hunk 3 above)
Comment (reebhub) [reply Documentation/6.0/Raven.Documentation.Pages/sharding/unsupported.markdown:60]: leaving it in for now, waiting for an update..

**Pull request merged**